TYPO3.Neos/Resources/Private/Partials/Module/Management/Workspaces/ContentChangeDiff.html

```diff
@@ -3,19 +3,27 @@
 <div class="neos-content-diff">
     <table class="neos-content-diff">
         <tr>
-            <th title="{f:if(condition: change.node.nodeType.label, then: '{neos:backend.translate(id: change.node.nodeType.label)}', else: '{change.node.nodeType.name}')}" data-neos-toggle="tooltip">
+            <th>
                 <f:if condition="{change.configuration.ui.icon}">
-                    <i class="{change.configuration.ui.icon}"></i>
+                    <i class="{change.configuration.ui.icon}" title="{f:if(condition: change.node.nodeType.label, then: '{neos:backend.translate(id: change.node.nodeType.label)}', else: '{change.node.nodeType.name}')}" data-neos-toggle="tooltip"></i>
                 </f:if>
                 {change.node.label}
             </th>
+            <f:if condition="{contentDimensions}">
+                <th>
+                    <f:render partial="Module/Shared/NodeContentDimensionsInformation.html" arguments="{contentDimensions: contentDimensions, node: change.node.nodeData}" />
+                </th>
+            </f:if>
+            <th>
+                <span title="{neos:backend.translate(id: 'lastModification')}: {change.node.lastModificationDateTime -> f:format.date(format: 'Y-m-d H:i')}" data-neos-toggle="tooltip">{change.node.lastModificationDateTime -> m:format.relativeDate()}</span>
```

Comment:
We should show here at least a label what kind of time is shown here. As an editor i'm not sure if it is the time when published, modified, created or whatever 
![screen shot 2016-04-08 at 09 07 49](https://cloud.githubusercontent.com/assets/769789/14376406/8f93ae76-fd69-11e5-8597-63623386f7af.png)